public/data/chapters/ar.json

```diff
@@ -3,684 +3,798 @@
         "revelationPlace": "makkah",
         "transliteratedName": "الفاتحة",
         "versesCount": 7,
-        "translatedName": "الفاتحة"
+        "translatedName": "الفاتحة",
+        "slug": "سورة-الفاتحة"
     },
     "2": {
         "revelationPlace": "madinah",
         "transliteratedName": "البقرة",
         "versesCount": 286,
-        "translatedName": "البقرة"
+        "translatedName": "البقرة",
+        "slug": "البقرة-سورہ"
     },
     "3": {
         "revelationPlace": "madinah",
         "transliteratedName": "آل عمران",
         "versesCount": 200,
-        "translatedName": "آل عمران"
+        "translatedName": "آل عمران",
+        "slug": "آل-عمران"
     },
```

Comment:
quick one @osamasayed ,

I am not very good at arabic grammar. But is it normal that
1. surah - al fatihah
2. al-baqarah- surah
3. Ali Imran

Reply:
The data is fetched from BE.